tests/wallet/nft_wallet/test_nft_wallet.py

```diff
@@ -1261,6 +1262,8 @@ async def test_nft_set_did(two_wallet_nodes: Any, trusted: Any) -> None:
         time_left -= 0.5
     nft_wallet_1_id = coins_response.get("wallet_id")
     assert nft_wallet_1_id
+    assert len(wallet_node_0.wallet_state_manager.wallets[nft_wallet_0_id].my_nft_coins) == 0
```

Comment:
is this checking that old wallet doesn't have coins anymore?

Reply:
yes